### ISSUE TITLE
Outputting BUILD_LOG in the Failed message.

### ISSUE THREAD

**Issue by njt1982** [jenkinsci/ghprb-plugin#555]
Is it possible to send `$BUILD_LOG` to the Build Status Message? That way if something fails (eg linting), the errors are present in the PR...

**Comment by njt1982** [comment]
Seems to relate to (or possibly even duplicate) #542 ?

**Comment by manovotn** [comment]
Yep, looks like a copy of what I tried to achieve (and failed). Have you managed to find a way?

**Comment by njt1982** [comment]
Nope :( I wonder if @benpatterson  knows?

**Comment by manovotn** [comment]
I worked around this limitation by manually calling GH API and updating the status. You need secondary Jenkins job which will be triggered after your PR builder (else the result will be overwritten). You can pass over the variables from this plugin to the other job (like PR SHA). It is messy solution but gets the job done.
